### DISCUSSION TITLE
如何让openclash只代理从代理端口来的流量

### DISCUSSION THREAD

**Discussion by eidoid** [vernesong/OpenClash#4204] [Q&A] [answered]
就像普通clash软件一样，不代理普通经过路由器的流量，而只关注OmegaSwitch或任何其他方式手动设置代理来的流量

**Comment by fejeIaqbhbv** [comment, marked as answer]
关dns劫持然后本地绕过地址填0.0.0.0/0？
复杂点的方法也有
反正没有只启动内核不添加任何防火墙规则的方法，估计设计时没考虑
或者换软件
